src/utils/toSQL.js

```diff
@@ -397,6 +397,71 @@ export function jsonToSQLite(obj) {
     .join("\n");
 }
 
+export function jsonToDjangoModels(obj) {
+  let djangoCode = "";
+
+  obj.types.forEach(type => {
+    let typeStatements = [];
+    
+    type.fields.forEach(field => {
+      if (field.type === "ENUM" || field.type === "SET") {
+        typeStatements.push(`    ${field.name}_t = models.TextChoices(\n${field.values.map(value => `        ('${value}', '${value}')`).join(',\n')}\n    )`);
+      }
+    });
+
+    // Generate Django model definition
+    djangoCode += `class ${type.name}(models.Model) {\n`;
+    type.fields.forEach(field => {
+      djangoCode += `    ${field.name} = models.${getTypeString(field)}(`;
```

Comment:
`getTypeString(field)` will get the type in a given DBMS, which by default is MySQL, so e.g. this will result in `models.VARCHAR` instead of `models.CharField`.

The necessary conversions from `sqlDataTypes` in `constants.js` to django data types should be added.

Reply:
Resolved